docs/en/operation/Scalability-Tuning.md

```diff
@@ -8,20 +8,114 @@ priority: 9
+* Latency of journal replay - The journal replay, which is the majority of the cold startup time for
+a master, takes time proportional to the number of files in the system.
+* Latency of journal backup - The journal backup takes time proportional to the number of files in
+the system. If delegated backups are used, the primary master will not be impacted during the entire
+backup duration.
+* Latency of recursive operations - Recursive operations such as `loadMetadata` and `delete` take
+time proportional to the number of files in the subtree being operated on. This should not impact
+user experience unless the subtree is significantly large (> 10k files).
+
+### Number of Concurrent Clients
+
+Concurrent clients represents number of logical Alluxio clients communicating with the Alluxio
+Master or Worker. Concurrency is typically considered at a per node level.
+
+Calculating concurrent clients requires estimating the number of Alluxio clients in the deployment.
+This can typically be attributed to the number of threads allowed in the compute frameworks used.
+For example, the number of tasks in a Presto job or the number of slots in a MapReduce node.
+
+#### Master
+
+Client connections to the master are short lived, so we first estimate the number of concurrent
+clients and then convert to a operations/second metric
+* 1 per Alluxio worker in the cluster
+* Max of
+	* Number of concurrent clients of the system as calculated above
+	* (alluxio.worker.block.master.client.pool.size + alluxio.user.file.master.client.pool.size.max)
+	per user per service using the Alluxio client
+
+For example, in a deployment with 2 users, 50 Presto worker nodes (with 200 task concurrency), and
+50 Alluxio nodes, the estimations would come out to the following
+* 50 (workers) + 2 (users) x 11 (block pool size) x 10 (file pool size) x 50 (services) = 11050
+* 50 (workers) + 50 (Presto workers) x 200 (task concurrency) = 10050
+
+Yielding a max of 11050
+
+Note based on the number of concurrent queries, add 1 more service for each to account for the
+Presto coordinator’s connections.
+
+If a maximum latency of 1 second is expected at absolute peak capacity, the master would need to
+support about 11050 operations per second. The typical operation to benchmark is `getFileInfo` for
+OLAP frameworks. Note that although the number of potential concurrent clients are high, it is
+unlikely for all clients to simultaneously hit the master. The steady state number of concurrent
+clients to the master is generally much lower.
+
+The number of concurrent clients to the master impacts the following
+* Number of cores required by the master - We recommend 8 clients per core, or to determine the
+number of cores based on required operation throughput.
+* Number of open files allowed on the master - We recommend about 4 open files per expected
+concurrent client. On Linux machines this can be set through the `ulimit` command.
```

Comment:
The 'ulimit' command generally only allows viewing the limits. While it can modify the limits within a session, Alluxio start scripts may use loopback ssh which will not respect changed limits. I would modify this to suggest that the values be set through '/etc/security/limits.d/'